internal/commands/relationship.go

```diff
@@ -250,100 +270,203 @@ func readRelationships(cmd *cobra.Command, args []string) error {
 	}
 }
 
+func argsToRelationship(args []string) (*v1.Relationship, error) {
```

Comment:
It looks like we spotted using this in favor of tupleToRel, but this does additional validation because `tuple.ParseRel` checks for valid/invalid characters. We should unify the usage and make sure we're using the `tuple.ParseRel`

Reply:
good catch!